priv/templates/extended_bin

```diff
@@ -343,6 +343,24 @@ case "$1" in
              "unpack" "$REL_NAME" "$NAME_TYPE" "$NAME" "$COOKIE" "$2"
         ;;
 
+    uninstall)
```

Comment:
It is true that these is similar, but unpack also similar.
And, I think that we should divide the main function (at install_upgrade_escript), because the inside of the processing is because different.

Although it is possible to put together about here, should I also merge the unpack?